### ISSUE TITLE
Can't import DistanceMatcher

### ISSUE THREAD

**Issue by 5100117** [wannesm/LeuvenMapMatching#18]
Hi,
I am trying to run the example, but I got this error. Do you have any suggestion to this error?
the error is.
```Traceback (most recent call last):
  File "test_match.py", line 2, in <module>
    from leuvenmapmatching.matcher.distance import DistanceMatcher
  File "/home/aa/aa/py37/lib/python3.7/site-packages/leuvenmapmatching/matcher/distance.py", line 13, in <module>
    from .base import BaseMatching, BaseMatcher
  File "<fstring>", line 1
    (prune_thr=)
              ^
SyntaxError: invalid syntax
```

And here is my example code.
```
from leuvenmapmatching import visualization as mmviz
from leuvenmapmatching.matcher.distance import DistanceMatcher
from leuvenmapmatching.map.inmemmap import InMemMap

map_con = InMemMap("mymap", graph={
    "A": ((1, 1), ["B", "C", "X"]),
    "B": ((1, 3), ["A", "C", "D", "K"]),
    "C": ((2, 2), ["A", "B", "D", "E", "X", "Y"]),
    "D": ((2, 4), ["B", "C", "D", "E", "K", "L"]),
    "E": ((3, 3), ["C", "D", "F", "Y"]),
    "F": ((3, 5), ["D", "E", "L"]),
    "X": ((2, 0), ["A", "C", "Y"]),
    "Y": ((3, 1), ["X", "C", "E"]),
    "K": ((1, 5), ["B", "D", "L"]),
    "L": ((2, 6), ["K", "D", "F"])
}, use_latlon=False)

path = [(0.8, 0.7), (0.9, 0.7), (1.1, 1.0), (1.2, 1.5), (1.2, 1.6), (1.1, 2.0),
        (1.1, 2.3), (1.3, 2.9), (1.2, 3.1), (1.5, 3.2), (1.8, 3.5), (2.0, 3.7),
        (2.3, 3.5), (2.4, 3.2), (2.6, 3.1), (2.9, 3.1), (3.0, 3.2),
        (3.1, 3.8), (3.0, 4.0), (3.1, 4.3), (3.1, 4.6), (3.0, 4.9)]

matcher = DistanceMatcher(map_con, max_dist=2, obs_noise=1, min_prob_norm=0.5)
states, _ = matcher.match_incremental(path[:5])
states, _ = matcher.match_incremental(path[5:], backtrace_len=-1)
nodes = matcher.path_pred_onlynodes

mmviz.plot_map(map_con, matcher=matcher,
               show_labels=True, show_matching=True, show_graph=True,
               filename="my_plot.png")
```

Thanks for your help.

**Comment by wannesm** [comment]
That's a Python 3.8 feature for f-strings that sneaked in. I have uploaded a new version (1.1.1) without this syntax to restore 3.7 compatibility.

**Comment by 5100117** [comment]
thank you.
I installed version 1.1.1 but I got another error.
```
Searching closeby nodes with linear search, use an index and set max_dist
Traceback (most recent call last):
  File "test_match.py", line 24, in <module>
    states, _ = matcher.match_incremental(path[:5])
  File "/home/aa/aa/py37/lib/python3.7/site-packages/leuvenmapmatching/matcher/base.py", line 672, in match_incremental
    if self.non_emitting_states and not self._skip_ne_states(obs_idx):
  File "/home/aa/aa/py37/lib/python3.7/site-packages/leuvenmapmatching/matcher/distance.py", line 262, in _skip_ne_states
    if next_ne_m.d_s > 0:
AttributeError: 'int' object has no attribute 'd_s'
```

**Comment by wannesm** [comment]
This was due to incremental matching still using an old datastructure. This is fixed in 91ca969fceadea79eb1fba37f66686b52054f187 (see updated doc for an example: https://leuvenmapmatching.readthedocs.io/en/latest/usage/incremental.html)
